test_framework/scripts/longhorn-setup.sh

```diff
@@ -399,16 +398,37 @@ main(){
       get_rancher_api_key
       install_longhorn_by_rancher
     fi
-    run_longhorn_tests ${WORKSPACE}
+    run_longhorn_tests
   elif [[ "${LONGHORN_UPGRADE_TEST}" == true || "${LONGHORN_UPGRADE_TEST}" == True ]]; then
```

Comment:
Why there are two same conditions? `elif [[ "${LONGHORN_UPGRADE_TEST}" == true || "${LONGHORN_UPGRADE_TEST}" == True ]]; then`

Reply:
removed the duplicated one.